File: C_Programming/Study/Q1_Assignment30.c
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name :  DisplayASCII
// Description:     Assignment_30: Q.1 :Write a program whih displays ASCII table. Table contains symbol, Decimal,
//                  Hexadecimal and Octal representation of every member from 0 to 255.
//          
// Date:            23/06/2026
// Author:          Shital Ajit Nikam   
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

void DisplayASCII()
{
    int iCnt = 0;
    printf("-------------------------------------------------\n");
    printf("| Decimal  | Hexadecimal  | Octal  | Symbol  |\n");
    printf("-------------------------------------------------\n");

    for(iCnt = 0; iCnt <= 255; iCnt++)
    {
        printf("|  %7d  | %10X  |  %4o  |  %7c   |\n",iCnt,iCnt,iCnt,iCnt);
    }

    printf("-------------------------------------------------\n");
    
}

int main()
{

    DisplayASCII();
    
    return 0;
}

File: Python_Programming/Competative/Assignment30/Q1_Assignment30.py
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
#        
# Description: Assignment_30: Q.1: write a program which prints "Jay Ganesh..." every 2 seconds.                    
#                                     
# Date:            20/07/2026
# Author:          Shital Ajit Nikam   
#
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import schedule


#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
# Function Name: Display: Print Jay Ganesh for every 2 seconds on screen.
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////

def Display():                                          # call back function , task logic
    print("Jay Ganesh...")
    
def main():
    Border = "-"*60
    print(Border) 
    print(" Marvellous Automation Script ")
    print(Border) 
    
    schedule.every(2).seconds.do(Display)               #call Display method.
    
    while True:
        schedule.run_pending()          # will check pending tasks explicitely if missed any tasks
           
            
    print(Border) 
    print(" Thank you for using Marvellous Automation Script ")
    print(Border) 
        
    
if __name__ == "__main__":
    main()